providers/json.js

```diff
@@ -54,6 +55,16 @@ exports.getAll = (table) => {
 exports.get = (table, document) => fs.readJSON(resolve(baseDir, table, `${document}.json`)).catch(() => null);
 
 /**
+ * Check if the document exists.
+ * @param {string} table The name of the directory.
+ * @param {string} document The document name.
+ * @returns {Promise<boolean>}
+ */
+exports.has = (table, document) => fs.access(resolve(baseDir, table, `${document}.json`))
```

Comment:
Ao pls, it's literally the same thing. https://github.com/bdistin/fs-nextra/blob/master/index.js#L146